core-bundle/src/Routing/Page/PageRoute.php

```diff
@@ -130,7 +130,7 @@ public function setContent(mixed $content): self
         return $this;
     }
 
-    public function getContent()
+    public function getContent(): ?object
```

Comment:
```suggestion
    public function getContent(): object|null
```

Does this work, too?